tests/playwright-tests/tests/adminandpublic/publishPublication.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test } from '@playwright/test';
+import environment from '../../utils/env';
+import AzureLoginPage from '../../admin/azpage/AzureLoginPage';
+import AdminPage from '../../admin/pages/AdminPage';
+import CreateNewPublicationPage from '../../admin/pages/CreateNewPublicationPage';
+import uiTestString from '../../utils/uiTestString';
+import ContentPage from '../../admin/pages/ContentPage';
+import ManagePublicationReleasePage from '../../admin/pages/ManagePublicationReleasePage';
+import HomePage from '../../general-public/pages/HomePage';
+import FindStatisticsPage from '../../general-public/pages/FindStatisticsPage';
+import CreateNewReleasePage from '../../admin/pages/CreateNewReleasePage';
+import SingOffPage from '../../admin/pages/SignOffPage';
+
+test.describe.configure({ mode: 'serial' });
+
+let adminPage: AdminPage;
+let azPage: AzureLoginPage;
+let createNewPublicationPage: CreateNewPublicationPage;
+let managePublicationReleasePage: ManagePublicationReleasePage;
+let contentPage: ContentPage;
+let homePage: HomePage;
+let createNewReleasePage: CreateNewReleasePage;
+let signOffPage: SingOffPage;
+let findStatisticsPage: FindStatisticsPage;
+let uiTestPublicationName: string;
+let uiTestPublicationSummary: string;
+
+test.beforeEach(async ({ page }) => {
+  adminPage = new AdminPage(page);
+  azPage = new AzureLoginPage(page);
+  createNewPublicationPage = new CreateNewPublicationPage(page);
+  managePublicationReleasePage = new ManagePublicationReleasePage(page);
+  contentPage = new ContentPage(page);
+  homePage = new HomePage(page);
+  createNewReleasePage = new CreateNewReleasePage(page);
+  signOffPage = new SingOffPage(page);
+  findStatisticsPage = new FindStatisticsPage(page);
+});
+
+test('Verify that user is able to create a release via admin', async ({
+  page,
+}) => {
+  await page.goto(environment.ADMIN_URL);
+  await adminPage.clickSignIn();
+  await azPage.doSignIn();
+  await page.waitForLoadState('load');
+  adminPage.selectTestThemeAndTestTopic();
+  await adminPage.createNewPublicationButton.click();
+
+  uiTestPublicationName = uiTestString('publish publication');
```

Comment:
I'd shorten these down to `publicationName` and `publicationSummary` personally - makes them a little easier to read and I know what they mean still.  Just my personal preference.